frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -141,19 +141,22 @@ export const FIXED_WIDTH = "1048px";
 export const FixedWidthContainer = styled.div<{
   isFixedWidth: boolean;
 }>`
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
   width: 100%;
 
-  ${breakpointMaxSmall} {
-    flex-direction: column;
-  }
-
   ${({ isFixedWidth }) =>
     isFixedWidth &&
     css`
       margin: 0 auto;
       max-width: ${FIXED_WIDTH};
     `}
 `;
+
+export const StyledFixedWidthContainer = styled(FixedWidthContainer)`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+
+  ${breakpointMaxSmall} {
+    flex-direction: column;
+  }
+`;
```

Comment:
`StyledFixedWidthContainer` seems to be used only for parameters. WDYT about stating that explicitly via the component name? E.g. it could be `ParametersFixedWidthContainer`